cmd/minikube/main.go

```diff
@@ -205,7 +205,7 @@ func setFlags(parse bool) {
 		if pflag.CommandLine.Changed("log_dir") && pflag.Lookup("log_dir").Value.String() != "" {
 			dir = pflag.Lookup("log_dir").Value.String()
 		}
-		l := logFileName(dir, 0)
+		l := logFileName(dir, 0, pflag.Args())
```

Comment:
`pflag` is a global variable, so you can avoid having to pass it as an argument to `logFileName`.

Reply:
Thanks for the review, I have made the changes

`nonFlagArgs := pflag.Args()`
`l := logFileName(dir, 0, nonFlagArgs)`
in the new commit.

Regards

Reply:
That's not exactly what I meant, this still work perfectly fine though.

What I'm trying to communicate is that because `pflag` is a global variable, it doesn't need to be passed into `logFileName`, be can accessed within because it's global.

So you can revert the signature back to `l := logFileName(dir, 0)`

And then before `for _, s := range nonFlagArgs {` you can do `nonFlagArgs := pflag.Args()`

Let me know if that makes sense.

Reply:
Hi. I have fixed this in the latest commits